x-pack/test/security_solution_endpoint/apps/endpoint/resolver.ts

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const pageObjects = getPageObjects(['common', 'timePicker', 'hosts', 'settings']);
+  const testSubjects = getService('testSubjects');
+  const esArchiver = getService('esArchiver');
+
+  describe('Endpoint Event Resolver', function () {
+    this.tags(['ciGroup7']);
+
+    before(async () => {
+      await esArchiver.load('endpoint/resolver_tree/api_feature', { useCreate: true });
+      await pageObjects.hosts.navigateToSecurityHostsPage();
+      const fromTime = 'Jan 1, 2018 @ 00:00:00.000';
+      const toTime = 'now';
+      await pageObjects.timePicker.setAbsoluteRange(fromTime, toTime);
+      await (await testSubjects.find('draggable-content-host.name')).click();
+      await testSubjects.existOrFail('header-page-title');
+      await (await testSubjects.find('navigation-events')).click();
+      await testSubjects.existOrFail('events-viewer-panel');
+      await (await testSubjects.findAll('investigate-in-resolver-button'))[0].click();
+      await testSubjects.exists('investigate-in-resolver-button', { timeout: 4000 });
+    });
+
+    it('check that Resolver and Data table is loaded', async () => {
+      await testSubjects.existOrFail('resolver:graph');
+      await testSubjects.existOrFail('tableHeaderCell_name_0');
+      await testSubjects.existOrFail('tableHeaderCell_timestamp_1');
+    });
+
+    it('compare resolver Nodes Table data and Data length', async () => {
+      const nodeData: string[] = [];
+      const TableData: string[] = [];
+
+      const Table = await testSubjects.findAll('resolver:node-list:item');
+      for (const value of Table) {
+        const text = await value._webElement.getText();
+        TableData.push(text.split('\n')[0]);
+      }
+
+      await testSubjects.click('resolver:graph-controls:zoom-out');
```

Comment:
It there a reason to zoom out and back in for this test?

Reply:
Yes I getting back to the original state

Reply:
Is it necessary to zoom out for this test?  Is that dependent on the data set or why zoom out in this test when you're comparing nodes to table data?

Reply:
yes because when we have a big tree some times it will not see the all the nodes